frontend/public/components/_toggle-play.scss

```diff
@@ -6,6 +6,13 @@
   width: 32px;
   padding: 0;
 
+  &:disabled {
+    border-color: $color-grey-border !important;
+    &:before {
+      content: '\f05e' !important;
```

Comment:
Here and on line 17 it'd be best to use the SASS var. https://github.com/FortAwesome/Font-Awesome/blob/v4.7.0/scss/_variables.scss#L66